tests/unit/test_inference.py

```diff
@@ -257,3 +265,63 @@ def test_apply_on_patches_window():
     )
 
     assert out is not None and out.shape == (1, 3, 12 * 1024**2)
+
+
```

Comment:
too much copy paste. There are now 4 tests in this file which look nearly identical. You should add some arguments to the existing tests, and use pytest.mark.parametrize. Seems like you need to parametrize over torch_compile and distilled vs not.


```
def  test_cbottle3d_sample(time_stepper, channels_last, torch_compile, distilled):
     model = create_distilled() if distilled else create_not_distilled()
     if torch_compile:
          # torch copmile stuff
```